scripts/reports/comparison-notebook.Rmd

```diff
@@ -40,23 +40,29 @@ library(purrr)
 ```
 
 ```{r}
+# outside_forecaster_subset <- c("COVIDhub-baseline", "COVIDhub-trained_ensemble", "COVIDhub_CDC-ensemble")
+# i <- 1
+# forecaster_families <- tar_read(forecaster_params_grid)$family %>% unique()
+# forecaster_family <- forecaster_families[[i]]
+# params_subset <- tar_read(forecaster_params_grid) %>% filter(family == forecaster_family)
```

Comment:
I'm kind of surprised these tar_reads don't cause problems. I guess this is getting run in a fresh instance of R?

Reply:
These are comments purely for my own REPL debugging of these notebooks. It's hard to step into the notebook and inspect what's breaking otherwise.